platform/viewer/src/connectedComponents/ConnectedExitPluginSwitch.js

```diff
@@ -0,0 +1,64 @@
+import OHIF from "@ohif/core";
+import ExitPluginSwitch from "./ExitPluginSwitch.js";
+import { connect } from "react-redux";
+
+const { setLayout } = OHIF.redux.actions;
+
+const mapStateToProps = state => {
+  const { activeViewportIndex, layout, viewportSpecificData } = state.viewports;
+
+  return {
+    activeViewportIndex,
+    viewportSpecificData,
+    layout
+  };
+};
+
+const mapDispatchToProps = dispatch => {
+  return {
+    setLayout: data => {
+      dispatch(setLayout(data));
+    }
+  };
+};
+
+const mergeProps = (propsFromState, propsFromDispatch, ownProps) => {
+  const { layout: currentLayout } = propsFromState;
+  const { setLayout } = propsFromDispatch;
+  const { togglePlugSwitchStatus } = ownProps;
+  const exitMpr = () => {
+    let viewports = [];
+    const rows = 1;
+    const columns = 1;
+
+    const viewport = currentLayout.viewports[0];
+    let plugin = viewport && viewport.plugin;
+    if (viewport && viewport.vtk) {
+      plugin = 'cornerstone';
+    }
+    viewports.push({
+      height: `${100 / rows}%`,
+      width: `${100 / columns}%`,
+      plugin,
+    });
+
+    const layout = {
+      viewports,
+    };
+
+    setLayout(layout);
```

Comment:
I believe setLayout takes:

```js
{
  numRows: 1,
  numColumns: 1,
  viewports: [{ plugin: 'plugin-type' }],
}```

Reply:
I just borrowed the code related to setLayout from ConnectedLayoutButton.js as it checked for `currentLayout.viewports[i]` and then set the plugin. If you suggest that this is unnecessary and something like below is sufficient, I will make that change.
```
{
  numRows: 1,
  numColumns: 1,
  viewports: [{ plugin: 'cornerstone' }],
}
```